src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md

```diff
@@ -8,28 +8,32 @@ A specific handler component is deployed for the
 [OVN](https://www.ovn.org/en/architecture/) is a project that builds on top
 of Open vSwitch providing a rich high level API for describing virtual
 network components like Logical Routers, Logical Switches, Load balancers,
-Logical Ports. OVN Kubernetes is a Cloud Management System Plugin (CMS plugin)
-in terms of the OVN project.
+Logical Ports. [OVN Kubernetes](https://github.com/ovn-org/ovn-kubernetes) is a Cloud Management System Plugin (CMS plugin)
+which manages OVN resources to setup networking for Kubernetes clusters.
 
 The OVN Kubernetes handler watches for Submariner `Endpoints` and Kubernetes
 `Nodes` and interfaces with the OVN databases (OVN NorthDB and SouthDB) to store and
-maintain information necessary for Submariner, including:
+create OVN resources necessary for Submariner, including:
 
 * A logical router named `submariner_router` that handles the communication
   to remote clusters and has a leg on the network which can talk to the
-  `ovn-k8s-gw0` interface on the Gateway node. This router is pinned to
+  `ovn-k8s-sub0` interface on the Gateway node. This router is pinned to
   the active Gateway chassis.
 
-* Routing policies added to the existing `ovn_cluster_router` which redirect
+* OVN Logical Router Static Routes added to the `submariner_router` to ensure
+  local traffic destined for remote clusters and remote traffic destined for local
+  resources is routed correctly.
+
+* OVN Logical Router Policies added to the existing `ovn_cluster_router` which redirect
   traffic targeted for remote routers through the `submariner_router`.
 
 * A `submariner_join` logical switch that connects the `submariner_router`
   with the `ovn_cluster_router`.
 
 ## The handler architecture
 
-The following diagram illustrates the OVN Kubernetes handler architecture where the
-blue elements represent the OVN Kubernetes native network elements and the yellow
-elements are introduced by Submariner.
+The following diagram illustrates the required Submariner OVN architecture overlayed
+on the native OVN-Kubernetes managed OVN architecture and components. The specific
+`networkpluginsyncer` managed OVN components are boxed in green.
 
-![Submariner with OVNKubernetes architecture](/images/ovn-kubernetes/submariner-on-ovn-v2.svg)
+![Submariner with OVNKubernetes architecture](/images/ovn-kubernetes/ovn-submariner-architecture.svg)
```

Comment:
Nice diagram with explanation @astoycos, thanks.

Reply:
@astoycos In the attached diagram, On OVN Submariner router, it is mentioned that "OVN Nat rules are applied here to ensure traffic is SNAT'ed to the globalnetIP when Globalnet is enabled".

Since this will be done when we support OVN with Globalnet and is not there currently, I'm not sure if we should capture this in the diagram. Basically, it might give an impression that Globalnet is supported even with OVN by looking into the diagram. What you do say?